src/Raven.Server/Smuggler/Documents/CsvStreamSource.cs

```diff
@@ -46,22 +70,36 @@ public class CsvStreamSource : ISmugglerSource, IDisposable
 
         private readonly List<IDisposable> _disposables = new List<IDisposable>();
 
-        public CsvStreamSource(DocumentDatabase database, Stream stream, DocumentsOperationContext context, string collection)
+        public CsvStreamSource(DocumentDatabase database, Stream stream, DocumentsOperationContext context, string collection, CsvConfiguration csvConfig)
         {
             _database = database;
             _stream = stream;
             _context = context;
             _currentType = DatabaseItemType.Documents;
             _collection = collection;
+            
+            _csvHelperConfig = new Configuration();
+            _csvHelperConfig.Delimiter = csvConfig.Delimiter;
+            _csvHelperConfig.Quote = csvConfig.Quote[0];
```

Comment:
why [0]?

Reply:
Updated `JsonDeserializationBase.GetValue` to accept char as well

Reply:
@Danielle9897
Can't we just use `QuoteString` instead of `Quote` ?

Reply:
we use quote because right now JsonDeserializationBase can handle chars

Reply:
As I understand, 
QuoteString is Not used to set the _'text qualifier character'._  (get only)
It is the string representation of the quote (text qualifier) character.

From the CSVHelper docs:

Quote:  
```
    /// <summary>
    /// Gets or sets the character used to quote fields.
    /// Default is '"'.
    /// </summary>
```

QuoteString:
```
    /// <summary>
    /// Gets a string representation of the currently configured Quote character.
    /// </summary>
```

Reply:
OK